src/context.rs

```diff
@@ -54,6 +56,11 @@ impl ContextBuilder {
         };
 
         let result = match root {
+            // SAFETY: [`wolfSSL_CTX_load_verify_buffer`][0] ([also][1]) is documented to get the valid `ctx` pointer from wolfSSL_CTX_new().
+            // It takes buffer with size specified as `sz` and `buf` is valid, since it is from a byte slice
```

Comment:
This sentence doesn't quite parse correctly.

Rather than grammar tweaks however I would suggest instead something like:

> The pointer given as the `in` argument must point to a region of `sz` bytes. The values passed here are valid since they are derived from the same byte slice.

(for all instances of the sentence of course)